src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1105,22 +1106,11 @@ impl Coordinator {
                         ty: "sink",
                     });
                 ctx.retire(Ok(ExecuteResponse::CreatedSink));
-                return;
             }
             Err(e) => {
                 ctx.retire(Err(e));
```

Comment:
Nit: If we need to call `ctx.retire` in every branch, we could pull it after the `match`, to make sure we never forget to do it. Also to remove a tiny amount of duplication.